xml/System.Diagnostics/Process.xml

```diff
@@ -2421,16 +2421,16 @@ You cannot cause processes on remote computers to exit. You can only view inform
 
  The process is terminating.</exception>
         <exception cref="T:System.NotSupportedException">You are attempting to call <see cref="M:System.Diagnostics.Process.Kill" /> for a process that is running on a remote computer. The method is available only for processes running on the local computer.</exception>
-        <exception cref="T:System.InvalidOperationException">The process has already exited.
+        <exception cref="T:System.InvalidOperationException">.NET Framework and .NET Core 3.0 and earlier versions only: The process has already exited.
 
 -or-
 
 There is no process associated with this <see cref="T:System.Diagnostics.Process" /> object.
 
 -or-
 
-The calling process is a member of the associated process' descendant tree.</exception>
-        <exception cref="T:System.AggregateException">Not all processes in the associated process' descendant tree could be terminated.</exception>
+The calling process is a member of the associated process's descendant tree.</exception>
```

Comment:
TIL something new. I always thought that only `s'` was valid. Seems that both variants of apostrophe for a word ending in "s" are equally valid: 

> For names that end in an s or z sound, though, you can either add -'s or just an apostrophe. Going with -'s is the more common choice:
> the car that belongs to Jones → Jones's car or Jones' car

Source: https://www.merriam-webster.com/words-at-play/what-happens-to-names-when-we-make-them-plural-or-possessive

Reply:
Yes, and MS style guide says to use 's too: https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28753